index.html

```diff
@@ -75,7 +75,7 @@ <h1>Im Lucca
                 <button type="button">javascript</button>
```

Comment:
This section should be created dynamically, please fix it.

<img width="1066" alt="Screen Shot 2022-09-15 at 15 29 05" src="https://user-images.githubusercontent.com/82426682/190502524-5e4b13dc-c9ad-4a73-9203-6f02587f4e44.png">